docs/standard/glossary.md

```diff
@@ -0,0 +1,265 @@
+---
+title: .NET Glossary
+description: Find out the meaning of selected terms used in the .NET documentation.
+keywords: .NET glossary, .NET dictionary, .NET terminology, .NET platform, .NET framework, .NET runtime
+author: tdykstra
+ms.author: tdykstra
+ms.date: 07/08/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# .NET Glossary
+
+The primary goal of this glossary is to clarify meanings of selected terms and acronyms that appear frequently in the .NET documentation without definitions.
+
+## AOT
+
+Ahead-of-time compiler.
+
+Similar to [JIT](#jit), this compiler also translates [IL](#il) to machine code. In contrast to JIT compilation, AOT compilation happens before the application is executed and is usually performed on a different machine. AOT tool chains don't trade runtime for compile time and thus can spend more time optimizing. Since the context of AOT is the entire application, the AOT compiler also performs cross-module linking and whole-program analysis, which means that all references are followed and a single executable is produced.
+
+## ASP.NET 
+
+The original ASP.NET implementation that ships with the .NET Framework.
+
+Sometimes ASP.NET is an umbrella term that refers to both ASP.NET implementations including ASP.NET Core. The meaning that the term carries in any given instance is determined by context. 
+
+See [ASP.NET](/aspnet/#pivot=aspnet).
+
+## ASP.NET Core
+
+A cross-platform, high-performance, open source implementation of ASP.NET built on .NET Core.
+
+See [ASP.NET Core](/aspnet/#pivot=core).
+
+## CLR
+
+Common language runtime.
+
+The exact meaning depends on the context, but this usually refers to the runtime of the .NET Framework. The CLR is a virtual machine that not only executes apps but also generates and compiles code on-the-fly using a JIT compiler. The current Microsoft CLR implementation is Windows only.
+
+## CoreCLR
+
+.NET Core common language runtime.
+
+This CLR is built from the same code base as the CLR. Originally, CoreCLR was the runtime of Silverlight and was designed to run on multiple platforms, specifically Windows and OS X. CoreCLR is now part of .NET Core and represents a simplified version of the CLR. It's still a cross platform runtime. CoreCLR is also a virtual machine with JIT and code execution capabilities.
+
+## CoreFX
+
+.NET Core framework.
+
+Also known as the .NET Core Base Class Library (BCL). A set of  System.* (and to a limited extent  Microsoft.\*) libraries. The BCL is a general purpose, lower-level frarmework that higher-level application frameworks, such as ASP.NET Core, build on. The source code of the .NET Core BCL is contained in the [CoreFX repository](https://github.com/dotnet/corefx). However, the majority of the .NET Core APIs are also available in the .NET Framework, so you can think of CoreFX as a fork of the .NET Framework BCL.
```

Comment:
I'd escape the asterisk on System.* just to be safe. It doesn't seem to confuse the markdown parser tho, so maybe these were ok as they were. Also, consider that normally we inline code these API references, so I wonder if they should be \`System.\*\` for `System.*` (no escape, of course) and \`Microsoft.\*\` for `Microsoft.*`.